app/controllers/categories_controller.rb

```diff
@@ -7,7 +7,8 @@ class CategoriesController < ApplicationController
   before_action :check_size, only: %i[new create edit update]
 
   def index
-    @pagy, @categories = pagy(Category.all.order(sort_params))
+    list = current_user&.admin_type? ? Category.all : Category.where(status: 'active')
```

Comment:
You also can use helper `Category.active` instead  `Category.where(status: 'active')`.

Reply:
Updated.